database/ServiceMySql.php

```diff
@@ -120,15 +156,30 @@ public function update($id, $data)
             echo "Resource successfully updated";
 
 
-        } catch (InvalidIdException $e){
+        } catch (InvalidIdException $e) {
             echo "Error: " . $e->getMessage();
         } catch (\Exception $e) {
             echo "Error updating resource: " . $e->getMessage();
         }
     }
 
+    /**
+     * Deletion of a resource
+     * @param $id
+     */
     public function delete($id)
     {
+        try {
+            $this->checkId($id);
+
+            $sql = "DELETE FROM guest.student WHERE id = " . $id;
```

Comment:
pls, always use prepare statements (same goes for the update() method)

Reply:
Done